src/components/modes/student/StudentView.js

```diff
@@ -95,8 +94,16 @@ export const StudentView = props => {
               });
             }
           }
+          const TIME_ZERO = 0;
+          const currentHour = Math.floor(
+            getTime() / ONE_MINUTE_IN_MILLIS / ONE_HOUR_IN_MINS,
+          );
+
           return (
-            <div className={classes.root}>
+            <div
+              style={{ visibility: !timerVisible ? 'visible' : 'hidden' }}
```

Comment:
I think you have the condition inverse here. If timer is not visible, then it's visible.